lib/omise/OmiseCharge.php

```diff
@@ -22,6 +22,18 @@ public static function retrieve($id = '', $publickey = null, $secretkey = null)
     }
 
     /**
+     * Search for charges.
+     *
+     * @param string query
+     *
+     * @return OmiseSearch
+     */
+    public static function search($query = '')
```

Comment:
@iwat right now we supported for the use cases,
```php
OmiseCharge::retrieve('id', $public_key, $secret_key);

// and

OmiseCharge::create($attrs, $public_key, $secret_key);
```

I think we need this one for search method also.